### DISCUSSION TITLE
page.meta.robots noindex if event.enddate < now

### DISCUSSION THREAD

**Discussion by wilbv** [derhansen/sf_event_mgt#1022] [Q&A] [answered]
Hi,

I am trying to remove detailpages of any events with (enddate < now) from my google index without hiding the event, since they are still needed for an archive listview.

My idea was to set noindex, nofollow on condition with TS:

```
[globalVar = TSFE:id = ##]

now = TEXT
now {
  data = date : U 
  #strftime = %d.%m.%Y
}

temp.eventDate = RECORDS
temp.eventDate {
  dontCheckPid = 1
  tables = tx_sfeventmgt_domain_model_event
  source.data = GP:tx_sfeventmgt_pievent|event
  source.intval = 1
  conf.tx_sfeventmgt_domain_model_event = TEXT
  conf.tx_sfeventmgt_domain_model_event {
    field = enddate
    if.value.field = enddate
    strftime = %d.%m.%Y
    if.isLessThan = now
  }
  page.meta.robots = noindex,nofollow
}

[global]
```

This is not working, I am kind of stuck.
Any help is appreciated, thank you.

**Comment by derhansen** [comment, marked as answer]
Try this in your `Detail.html` template:

```
<f:if condition="{event.enddate -> f:format.date(format: 'U')} < {f:format.date(date: 'now', format: 'U')}">
        <e:metaTag name="robots" content="noindex,nofollow" />
</f:if>
```

The condition compared the unix timestamp of the enddate of the event with the current unix timestamp and if the event enddate is reached, the additional meta tag is rendered.

**Reply by wilbv** [reply]
Thank you for answering.
Since we are still running Typo3 9.x and hence sf_event_mgt 4.x, this seems not supported. Any other approach maybe?

**Reply by derhansen** [reply]
No, sorry. Either update to v5 or v6 of the extension, copy the viewHelper to your own project or use a signal slot for the detail action and use the TYPO3 Meta Tag API to output the required meta data.
